### ISSUE TITLE
browser: keyboard not showing in fullscreen

### ISSUE THREAD

**Issue by daloes** [AndreyPavlenko/Fermata#212]
Haven't found issue for this so here it goes.
tldr: In car, keyboard is not showing, neither in android auto app on phone, when website is in fullscreen on Android Auto. Keyboard works in normal/fullscreen modes when running fermata on phone.

I tried multiple websites and this issue appears on all of them. If site is in normal view onscreen keyboard works as expected. If however, website goes fullscreen, keyboard no longer appears (clicking on any input does nothing, keyboard doesnt appear on screen. It doesn't even appear on phone in AA phone interface). 
If fullscreen is exited, keyboard works again normally. 

**This issue is only present when running Fermata on car infotainment**. If I run fermata on phone, keyboard works in both modes without issue so I'm not sure if this is fermata issue or some AA issue/limitation.
